package.json

```
@@ -0,0 +1,27 @@
+{
+  "name": "didcomm-messaging-clients-utilities",
+  "version": "1.0.0",
+  "type": "module",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "npm run test --workspaces",
+    "build": "npm run build --workspaces",
+    "format": "prettier --write .",
+    "format:check": "prettier --check ."
```

Add a master scripts entry for linting.
```suggestion
    "format:check": "prettier --check .",
    "lint": "npm run lint --workspaces"
```